### PR TITLE
feat: Add warning text for paid feature

### PR DESCRIPTION
**Pull request by shortdiv**
Add warning text with a prompt for users to confirm when accessing a paid feature

Screen Shot 2023-03-06 at 3 11 06 PM

### CODE REVIEW

**Review by tvdfly — COMMENTED** [review]
How about making a flag that can auto accept this? like `--yes` or `--auto-confirm`.

Alternatively, we could detect if we're in a non-interactive shell and assume yes since that's the existing behavior.

**Comment by shortdiv** [comment]
Oh, I like both of those ideas! I think creating a generic --yes flag would probably be the way to go with the least room for error

**Comment by shortdiv** [comment]
Updated!

File: internal/command/ips/allocate.go [hunk 1]
```diff
@@ -70,6 +76,15 @@ func runAllocateIPAddressV4(ctx context.Context) error {
 	if flag.GetBool(ctx, "shared") {
 		addrType = "shared_v4"
 	}
+	if !flag.GetBool(ctx, "yes") {
+		confirmIPV4, err := prompt.Confirm(ctx, "Looks like you're accessing a paid feature. Dedicated IPv4 addresses now costs $2/mo. Are you ok with this?")
+		if err != nil {
```
Comment (dangra) [inline internal/command/ips/allocate.go:81]: Standard way to check for non-interactive input is `prompt.IsNonInteractive(err)`

Comment (shortdiv) [reply internal/command/ips/allocate.go:81]: Oh neat til

**Review by dangra — APPROVED** [review]

**Pull request merged**